test/e2e/tests/solana/common-solana.ts

```diff
@@ -1543,103 +1542,6 @@ const featureFlagsWithSnapConfirmation = {
   },
 };
 
-async function startWebsocketMock(mockServer: Mockttp) {
```

Comment:
all this is removed:
- we start the solana ws server with mocks in our `withFixtures` function, where all the testing setup and tear down happens
- we have a separate file for our Solana mocks (all use the same)
- we re-route all the wss requests in our setupMock function